main/2021/CG-08-03.md

```diff
@@ -27,8 +27,10 @@ Installation is required, see the calendar invite.
     1. Review of action items from prior meeting.
     1. Update on JS API for Stack Switching (Ross/Luke/Francis) [10 mins]
         1. Poll for phase 1?
-    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [30 min].
+    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [50 min].
+        1. Poll for supporting [UTF-16](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
         1. Poll for maintaining single list-of-USV `string` type
+           * [Summary of concerns](https://github.com/WebAssembly/interface-types/issues/135#issuecomment-888878363) by W/UTF-16 languages
```

Comment:
I think this is still on the edge of editorialising, especially since this is @lukewagner's agenda item. It's not as though @lukewagner linked https://github.com/WebAssembly/interface-types/issues/135 as "summary of benefits".

@dschuff proposed discussing changes to this agenda item as a separate PR, which I think makes sense since we really want to make sure the UTF-16 vote is timetabled.

Reply:
I have done as I was told and do think that the context is both useful and reasonably stated.